Source/ManagedNativeWifi/NativeWifi.cs

```diff
@@ -1090,6 +1090,16 @@ internal static bool TryDetectBandChannel(uint frequency, out float band, out in
 				if (factor - Math.Floor(factor) == 0)
 					channel = (int)factor + 34;
 			}
+			else if (frequency is (>= 5_945_000 and <= 7_125_000))
+			{
+				// 6GHz
+				band = 6.0F;
+
+				var gap = frequency / 1_000M - 5_945M; // MHz
+				var factor = gap / 5M;
+				if (factor - Math.Floor(factor) == 0)
+					channel = Math.Max(1, (int)factor - 1);

```

Comment:
Allows the full range of frequencies considered 6GHz by the Windows CLI.

Frequencies below channel 1 (5955 MHz) are also considered channel 1. The Windows CLI does not currently use the channel and band determined here.